Makefile

```diff
@@ -171,6 +171,15 @@ test_sim_gaia_fast:
 	@echo "Running quick Gaia simulation. This may take several minutes..."
 	@go test ./cmd/gaia/app -run TestFullGaiaSimulation -SimulationEnabled=true -SimulationNumBlocks=500 -SimulationBlockSize=200 -SimulationCommit=true -SimulationSeed=10 -v -timeout 24h
 
+test_sim_gaia_import_export:
+	@echo "Running Gaia import/export simulation. This may take several minutes..."
+	@go test ./cmd/gaia/app -run TestGaiaImportExport -SimulationEnabled=true -SimulationNumBlocks=50 -SimulationBlockSize=200 -SimulationCommit=true -SimulationSeed=4 -v -timeout 24h
```

Comment:
Should we just throw this into a script (i.e. loop over desired seeds)?

Reply:
Agreed, since this was merged I'll fix in a separate PR.